srcpkgs/kgpg/template

```diff
@@ -13,10 +13,10 @@ makedepends="kwindowsystem-devel kcrash-devel kdbusaddons-devel
 short_desc="Simple interface for GnuPG, a powerful encryption utility"
 maintainer="k4leg <d0xi@inbox.ru>"
 license="GPL-2.0-only"
-homepage="https://kde.org/applications/utilities/kgpg/"
-distfiles="${KDE_SITE}/applications/${version}/src/${pkgname}-${version}.tar.xz"
-checksum=85bb9adbd7a7213bdd4913c2153582e938543db49cf2414b8794a850f63f3ab6
+homepage="https://kde.org/applications/utilities/$pkgname"
+distfiles="$KDE_SITE/release-service/$version/src/$pkgname-$version.tar.xz"
```

Comment:
Please use braces for variables, it's more readable and harder to use not what was meant.